packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -336,6 +339,12 @@ export class RunningSummarizer implements IDisposable {
                     timePending: Date.now() - this.heuristics.lastAttempted.summaryTime,
                 });
             });
+        summaryCollection.setPendingAckTimerTimeoutCallback(maxAckWaitTime, () => {
```

Comment:
@arinwt Some hot swap compat tests are failing saying setPendingAckTimerTimeoutCallback is not a function.
Isn't it all in runtime? How come it is affecting back compat tests?

Reply:
Ohh got it. The summary collection could be populated from previous state. So it can be from last version.